fairlearn/reductions/_moments/regression_moment.py

```diff
@@ -0,0 +1,134 @@
+from .utility_parity import UtilityParity
+from .moment import Moment,_LABEL,_GROUP_ID
+from .weighted_error_rate import WeightedErrorRate
+from .bounded_group_loss import SquareLoss
+import pandas as pd
+import numpy as np
+from itertools import repeat
+
+class CDF_DemographicParity(Moment):
+    short_name = "CDF_DemographicParity"
+
+    def __init__(self,loss,y_range,difference_bound=None,grids=[],grid_num=41):
```

Comment:
default 41 - any rationale?

Reply:
Actually not. But on the other hand, it should be some default number for this in the case when the user does not specify grids he wants to use. If you have any good idea about this, I am open to changes.

Reply:
I'm actually not so sure that's true. The thing with fairness is that we want people to make very deliberate choices. Perhaps @MiroDudik has thoughts.

You should, however, definitely remove the `grids` default parameter `[]` because of this: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Ok, I have changed the default parameter of `grids`.